curriculum/challenges/_meta/learn-html-forms-by-building-a-registration-form/meta.json

```
@@ -267,5 +267,10 @@
       "id": "60ffefd6479a3d084fb77cbc",
       "title": "Step 64"
     }
+    ,
+    {
+      "id": "6537e0be715fcb57d31ba8c3",
+      "title": "Step 65"
+    }
```

small formatting issue. the comma should be on line 269 so it matches the same pattern as the rest of the steps

```suggestion
    },
    {
      "id": "6537e0be715fcb57d31ba8c3",
      "title": "Step 65"
    }
```